tox.ini

```diff
@@ -5,7 +5,7 @@
 
 [tox]
 # pip is broken on py32
-envlist = py27, py30, py31, py33, py34, py35, py36, py37, py38
+envlist = py27, py30, py31, py33, py34, py35, py36, py37, py38, py39, py310
```

Comment:
Tools to run tests in various environment, including the py39 and py310

Reply:
Had installed Python 2.7 and run test against 2.7 (to ensure it still working), as well as sanity check from 3.7 - 3.10

Reply:
Those that doesn't have interpreter as I haven't installed that Python specific version. 
![image](https://user-images.githubusercontent.com/41770203/142782917-61a138d8-5295-485a-b41d-be9795bd1392.png)